docs/2.7.0/docs/daml-finance/tutorials/instrument-modeling/equity-extension.rst

```diff
@@ -145,21 +145,37 @@ components:
 - An option instrument, which describes the economic term of the rights a shareholder receives.
   For example, this could be a European option with a strike price below the current spot price, and
   a maturity three weeks in the future.
-  The :doc:`Generic Tutorial <generic-extension>` describes how to create a European option.
+  The :doc:`Option Tutorial <option-extension>` describes how to create a physically settled
+  European option.
 - The ``DeclareDividend`` choice to distribute the above option instrument in the correct proportion
   (e.g. 3 option contracts for each 10 shares held). This can be done in the same way as the Bonus
   Issue example described earlier, just change the ``perUnitDistribution`` line to distribute the
   option instrument you created above.
 
 When current shareholders receive the option instrument they can typically choose between:
 
-#. Exercising the option. The :doc:`Generic Tutorial <generic-extension>` describes how to elect to
-   exercise the option.
+#. Exercising the option. This corresponds to a Rights Subscription (described in more detail
+   in the next section below).
 #. Choosing not to exercise the option. The option will expire worthless.
 #. Selling the option. This is not always possible, it depends on the terms of the rights issue.
    :doc:`Getting Started: Settlement <../getting-started/settlement>` describes how this could be
    done.
 
+Rights Subscription
+*******************
+
+An investor can subscribe by electing to exercise the right (a call option), either in parts or in
```

Comment:
I didn't sleep great last night, but reading `subscribe by electing to exercise the right` made my head spin :) I wonder if there is a less verbose formulation

Reply:
In particular, shall we first introduce what a rights subscription is? Or is it already done elsewhere?